js/app/achilles.js

```diff
@@ -1,608 +1,706 @@
-(function () {
-	curl([
-		"jquery",
-		"d3",
-		"knockout",
-		"common",
-		"app/reports",
-		"bootstrap",
-		"d3/tip"
-	], function ($, d3, ko, common, reports) {
-		function summaryViewModel() {
-			var self = this;
-
-			self.dashboardData = ko.observable();
-			self.conditionsData = ko.observable();
-
-			self.personData = ko.observable();
-			self.observationPeriodsData = ko.observable();
-			self.datasource = ko.observable({
-				name: 'loading...'
-			});
-			self.datasources = [];
-
-			self.formatSI = function (d, p) {
-				if (d < 1) {
-					return d3.round(d, p);
+define(function(){
```

Comment:
Good! Yes, I see here, the define statement (and subsequent return of the achillesWeb object) makes this an AMD component.